src/Compilers/CSharp/Test/WinRT/Metadata/WinMdMetadataTests.cs

```diff
@@ -35,7 +35,7 @@ public class WinMdMetadataTests : CSharpTestBase
         /// System.Runtime.WindowsRuntime assembly.
         /// </summary> 
         [Fact]
-        public void FunctionPrototypeForwarded()
+        public void FunctionSignatureForwarded()
```

Comment:
>FunctionSignatureForwarded [](start = 20, length = 26)

Renamed so I could search for "prototype" to find places where issues need to be filed.